backend/.env

```
@@ -1,9 +1,9 @@
 NODE_ENV=production
 PORT=5000
 JWT_SECRET=yE9A$k8mP2xR7nQ4wL6sF3gH1vB5tN9cJ8dK2mY7pX4zA6qW3eR9sT1uI5oP8lM0
-MONGODB_URI=mongodb://localhost:27017/yega
-FRONTEND_URL=https://yega.3.85.74.100.nip.io
-EMAIL_USER=contacto@yega.com.mx
+MONGODB_URI=mongodb://localhost:27017/manda2
+FRONTEND_URL=http://localhost:8080
+EMAIL_USER=contacto@manda2.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_USER key should go before the FRONTEND_URL key

(UnorderedKey)